overlord/hookstate/ctlcmd/get.go

```diff
@@ -48,7 +48,7 @@ type getCommand struct {
 	Typed    bool `short:"t" description:"strict typing with nulls and quoted strings"`
 }
 
-var shortGetHelp = i18n.G("The get command prints configuration and interface connection settings.")
+var shortGetHelp = i18n.G("Print configuration options or interface connection settings")
```

Comment:
```suggestion
var shortGetHelp = i18n.G("Print either configuration options or interface connection settings")
```

This is a very subtle one (sorry), but I think because _interface_ can also be a verb adding `either` helps to set context.